tests/functional/gluster_stability/test_gluster_log_rotation.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import re
+
+import pytest
+
+from openshiftstoragelibs import baseclass
+from openshiftstoragelibs import exceptions
+from openshiftstoragelibs import openshift_ops
+from openshiftstoragelibs import waiter
+
+TCMU_CONF = "/etc/tcmu/tcmu.conf"
+TCMU_RUNNER_LOG = "/var/log/glusterfs/gluster-block/tcmu-runner.log"
+TCMU_ROTATE_LOG = "/etc/logrotate.d/gluster-block"
+LOG_REGEX = r"^(\d+-\d+-\d+.*\d+:\d+:\d+)\.\d+.*\[(\S+)\](.*):(\d+):(.*)$"
+
+
+class TestGlusterBlockLog(baseclass.GlusterBlockBaseClass):
+    """Class to validate logs for gluster block"""
+
+    def setUp(self):
+        super(TestGlusterBlockLog, self).setUp()
+        self.node = self.ocp_master_node[0]
+        self.g_node = self.gluster_servers[0]
+
+    def _set_log_level(self, g_node, level, msg, exec_time):
+        delete_log_level = r'sed -i "/\(^log_level.*=.*[0-9]\)/d" {}'
+        set_log_level = r'sed -i "\$alog_level = {}" {}'
+        check_log_msg = r'sed -n "/.*\({}\).*/{{p;}}" {} | tail -1'
+
+        # Set log level
+        openshift_ops.cmd_run_on_gluster_pod_or_node(
+            self.node, set_log_level.format(level, TCMU_CONF),
+            gluster_node=g_node)
+        self.addCleanup(
+            openshift_ops.cmd_run_on_gluster_pod_or_node,
+            self.node, delete_log_level.format(TCMU_CONF), gluster_node=g_node)
+
+        # Validate log level
+        log_msg = "log level now is {}".format(msg)
+        for w in waiter.Waiter(120, 3):
+            out = openshift_ops.cmd_run_on_gluster_pod_or_node(
+                self.node, check_log_msg.format(log_msg, TCMU_RUNNER_LOG),
+                gluster_node=g_node)
+            match = re.match(LOG_REGEX, out)
+            if (match
+                    and exec_time < datetime.datetime.strptime(
+                        match.group(1), "%Y-%m-%d %H:%M:%S")):
```

Comment:
`"%Y-%m-%d %H:%M:%S"` time format is used at multiple places, create variable for the same and use it in all places

Reply:
Done